pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/rbac/crd_editor_role.go

```diff
@@ -52,6 +53,11 @@ const crdRoleEditorTemplate = `# permissions for end users to edit {{ .Resource.
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
+    app.kubernetes.io/name: {{ lower .Resource.Kind }}-editor-role
+    app.kubernetes.io/component: crd-roles
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```

Reply:
updated in `kustomize/v1` plugin in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/8a2c1d7c48ec7743828b14a04ae9de765cde30b2

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.